styles/utils.go

```diff
@@ -128,3 +128,30 @@ func Seconds(value float64) string {
 func Milliseconds(value int) string {
 	return strconv.Itoa(value) + "ms"
 }
+
+func HSL(hue, saturation, lightness int) string {
+	var builder strings.Builder
+	builder.WriteString("hsl(")
+	builder.WriteString(strconv.FormatInt(int64(hue), 10))
+	builder.WriteString(",")
+	builder.WriteString(strconv.FormatInt(int64(hue), 10))
```

Comment:
This should be `saturation`

Reply:
My bad. I should have found this through test cases at least. Fixed and updated test cases.